client/browser/src/browser-extension/scripts/optionsPage.main.tsx

```diff
@@ -89,6 +89,10 @@ function requestGraphQL<T, V = object>(options: {
 const version = getExtensionVersion()
 const isFullPage = !new URLSearchParams(window.location.search).get('popup')
 
+// "Error code" constants for Sourcegraph URL validation
+export const URL_FETCH_ERROR = '0'
+export const URL_AUTH_ERROR = '1'
```

Comment:
I'd make these debuggable
```suggestion
export const URL_FETCH_ERROR = 'URL_FETCH_ERROR'
export const URL_AUTH_ERROR = 'URL_AUTH_ERROR'
```